tools/booster/Server.hs

```diff
@@ -138,9 +141,16 @@ main = do
                 mvarLogAction <- newMVar actualLogAction
                 let logAction = swappableLogger mvarLogAction
 
-                let defaultTactic = fromMaybe (SMT.List [SMT.Atom "check-sat-using", SMT.Atom "smt"]) koreSolverOptions.tactic
+                -- FIXME why do we need this? Does tactic == Nothing cause errors?
```

Comment:
It should be OK to give `Nothing`. In [Kore.SMT.defaultConfig](https://github.com/runtimeverification/haskell-backend/blob/0be07d293a1167615ef9342bb1820350f794e893/kore/src/SMT.hs#L442) we set the tactic to `Nothing` and use `List [Atom "check-sat"]` in [SimpleSMT.checkUsing](https://github.com/runtimeverification/haskell-backend/blob/0be07d293a1167615ef9342bb1820350f794e893/kore/src/SMT/SimpleSMT.hs#L723) if no CLI option has been given.

As I understand, `(check-sat)` does not mean that we're asking for the SAT, rather than SMT, solver. It's just the umbrella "do-your-best-I-dont-care-how" command.